presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -190,11 +190,51 @@ public void testBasicTypes()
                 .addRoundTrip("integer", "123456789", INTEGER)
                 .addRoundTrip("smallint", "32456", SMALLINT, "SMALLINT '32456'")
                 .addRoundTrip("tinyint", "5", SMALLINT, "SMALLINT '5'")
-                .addRoundTrip("double", "123.45", DOUBLE, "DOUBLE '123.45'")
-                .addRoundTrip("real", "123.45", REAL, "REAL '123.45'")
                 .execute(getQueryRunner(), prestoCreateAsSelect("test_basic_types"));
     }
 
+    @Test
+    public void testReal()
+    {
+        SqlDataTypeTest.create()
+                .addRoundTrip("real", "NULL", REAL, "CAST(NULL AS real)")
```

Comment:
It is not specific to this PR directly, but a second thought on the mechanism. Why express expected type as `spi.type.Type` instead a type name. Wouldn't having type names everywhere be more consistent?

Reply:
`Type` is more explicit (from Presto perspective). Also, it helps avoid having `String,String,String,String` signature, and helps adding overloads.